Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.17rc1-alpine
+ 
+# Creates an app directory to hold your app’s source code
+WORKDIR /app
+ 
+# Copies everything from your root directory into /app
+COPY . .
+ 
+# Installs Go dependencies
+RUN go mod download
+ 
+# Builds your app with optional configuration
+RUN go build -o /carsim
+ 
+# Tells Docker which network port your container listens on
+EXPOSE 8080
```

Comment:
Since this program does not listen to a port, exposing is unnecessary

Reply:
Absolutely, I'll get rid of it.

Reply:
Fixed in fa6979e986a937026be674ca017bb6e422d5320e and f604a5eca9a79a1bd5792f3232bb4621f645e3c4 (removed from config file)